02-peer-protocol.md

```diff
@@ -508,8 +523,11 @@ The funding node:
     - SHOULD send a `closing_signed` message.
 
 The sending node:
-  - MUST set `fee_satoshis` less than or equal to the
- base fee of the final commitment transaction, as calculated in [BOLT #3](03-transactions.md#fee-calculation).
+  - if `option_upfront_shutdown_script` applies to the final commitment transaction:
```

Comment:
The receiving node section also needs updating here?

Reply:
Good catch. I also used the wrong option name here, both fixed...